src/torcontrol.cpp

```diff
@@ -608,7 +608,9 @@ void TorController::disconnected_cb(TorControlConnection& conn)
     service = CService();
     if (!reconnect)
         return;
-    LogPrintf("tor: Disconnected from Tor control port %s, trying to reconnect\n", target);
+
+    LogPrint("tor", "tor: Disconnected from Tor control port %s, trying to reconnect\n", target);
```

Comment:
If we intend to keep this message unconditional, we should make it "Not connected to Tor control port" as @petertodd suggests in #7032. This doesn't imply being connected before.